package.json

```
@@ -3,12 +3,14 @@
   "version": "0.1.0",
   "private": true,
   "scripts": {
-    "dev": "next dev",
+    "dev": "npm run compile && next dev",
     "build": "next build",
     "start": "next start",
     "lint": "next lint",
     "storybook": "storybook dev -p 6006",
-    "build-storybook": "storybook build"
+    "build-storybook": "storybook build",
+    "compile": "graphql-codegen",
```

Can we give these slightly more descriptive names, something like? 
```suggestion
    "generate-gql-types": "graphql-codegen",
```